model-genome.txt

```diff
@@ -83,3 +83,31 @@ iML1515.json	pmid:29020004	ncbi_accession:NC_000913.3
_003621526.1,NW_003621538.1,NW_003621539.1,NW_003621541.1,NW_003621548.1,NW_003621549.1,NW_003621555.1,NW_003621562.1,NW_003621563.1,NW_003621567.1,NW_003621569.1,NW_003621573.1,NW_003621575.1,NW_003621592.1,NW_003621598.1,NW_003621607.1,NW_003621624.1,NW_003621660.1,NW_003621662.1,NW_003621674.1,NW_003621690.1,NW_003621701.1,NW_003621714.1,NW_003621715.1,NW_003621723.1,NW_003621752.1,NW_003621768.1,NW_003621780.1,NW_003621784.1,NW_003621791.1,NW_003621797.1,NW_003621800.1,NW_003621804.1,NW_003621830.1,NW_003621832.1,NW_003621838.1,NW_003621863.1,NW_003621869.1,NW_003621881.1,NW_003621894.1,NW_003621907.1,NW_003621932.1,NW_003621947.1,NW_003621948.1,NW_003621999.1,NW_003622004.1,NW_003622009.1,NW_003622034.1,NW_003622037.1,NW_003622038.1,NW_003622057.1,NW_003622061.1,NW_003622082.1,NW_003622088.1,NW_003622097.1,NW_003622118.1,NW_003622119.1,NW_003622126.1,NW_003622130.1,NW_003622136.1,NW_003622151.1,NW_003622155.1,NW_003622168.1,NW_003622173.1,NW_003622180.1,NW_003622184.1,NW_003622189.1,NW_003622214.1,NW_003622218.1,NW_003622239.1,NW_003622246.1,NW_003622251.1,NW_003622275.1,NW_003622291.1,NW_003622294.1,NW_003622314.1,NW_003622326.1,NW_003622330.1,NW_003622367.1,NW_003622368.1,NW_003622396.1,NW_003622403.1,NW_003622409.1,NW_003622419.1,NW_003622420.1,NW_003622446.1,NW_003622463.1,NW_003622480.1,NW_003622489.1,NW_003622502.1,NW_003622519.1,NW_003622536.1,NW_003622538.1,NW_003622540.1,NW_003622576.1,NW_003622587.1,NW_003622620.1,NW_003622643.1,NW_003622671.1,NW_003622727.1,NW_003622729.1,NW_003622762.1,NW_003622767.1,NW_003622768.1,NW_003622782.1,NW_003622788.1,NW_003622818.1,NW_003622827.1,NW_003622830.1,NW_003622840.1,NW_003622863.1,NW_003622869.1,NW_003622882.1,NW_003622899.1,NW_003622938.1,NW_003622961.1,NW_003622962.1,NW_003622967.1,NW_003622973.1,NW_003622983.1,NW_003623011.1,NW_003623013.1,NW_003623028.1,NW_003623044.1,NW_003623065.1,NW_003623076.1,NW_003623089.1,NW_003623096.1,NW_003623098.1,NW_003623113.1,NW_003623117.1,NW_003623151.1,NW_003623171.1,NW_003623186.1,NW_003623206.1,NW_003623240.1,NW_003623254.1,NW_003623287.1,NW_003623294.1,NW_003623304.1,NW_003623335.1,NW_003623344.1,NW_003623345.1,NW_003623351.1,NW_003623361.1,NW_003623375.1,NW_003623380.1,NW_003623402.1,NW_003623417.1,NW_003623439.1,NW_003623445.1,NW_003623452.1,NW_003623518.1,NW_003623529.1,NW_003623532.1,NW_003623536.1,NW_003623563.1,NW_003623564.1,NW_003623572.1,NW_003623579.1,NW_003623624.1,NW_003623633.1,NW_003623653.1,NW_003623659.1,NW_003623681.1,NW_003623688.1,NW_003623695.1,NW_003623696.1,NW_003623697.1,NW_003623716.1,NW_003623738.1,NW_003623742.1,NW_003623747.1,NW_003623763.1,NW_003623771.1,NW_003623775.1,NW_003623780.1,NW_003623785.1,NW_003623809.1,NW_003623812.1,NW_003623814.1,NW_003623865.1,NW_003623904.1,NW_003623937.1,NW_003623983.1,NW_003623992.1,NW_003624014.1,NW_003624049.1,NW_003624069.1,NW_003624073.1,NW_003624080.1,NW_003624135.1,NW_003624166.1,NW_003624169.1,NW_003624174.1,NW_003624179.1,NW_003624192.1,NW_003624267.1,NW_003624269.1,NW_003624313.1,NW
```

Comment:
Is there a new iCHO model? What's this line for?

Reply:
Yeah a new one for DG44 that reflects the loss of de novo purine synthesis so the additional req can enable selection of recomb clones